lib/pure/bitops.nim

```diff
@@ -68,55 +68,115 @@ when defined(nimHasalignOf):
   import macros
 
   type BitsRange*[T] = range[0..sizeof(T)*8-1]
-    ## Returns a range with all bit positions for type ``T``
+    ## Returns a range with all bit positions for type ``T``.
 
   proc setMask*[T: SomeInteger](v: var T, mask: T) {.inline.} =
-    ## Returns ``v``, with all the ``1`` bits from ``mask`` set to 1
+    ## Returns ``v``, with all the ``1`` bits from ``mask`` set to 1.
```

Comment:
set to 1.
=>
set to 1 via `bitor`

(otherwise user really wonders what's the difference with bitor)
ditto elsewhere.

Reply:
I don't think that's even a problem. Anyone that knows computer should know how bit masking work. We are just giving it a nice name here.